### ISSUE TITLE
Missing `statistics` when calling `Projects.show` with `{ statistics: true }`

### ISSUE THREAD

**Issue by NatoBoram** [jdalrymple/gitbeaker#3455]
**Description**

- Node.js version: v21.2.0
- Gitbeaker version: `^39.21.2`
- Gitbeaker release (cli, node, browser, core, requester-utils):
- OS & version: Pop!_OS 22.04 LTS x86_64 

**Steps to reproduce**

```ts
const project = await client.Projects.show("NatoBoram/switcheroo", {
	statistics: true,
}).catch((e: GitbeakerError) => {
	console.error(`Couldn't get project: ${e.message}`)
	exit(1)
})

// Property 'statistics' comes from an index signature, so it must be accessed with ['statistics'].
console.debug("project statistics", project.statistics)
```

**Expected behaviour**

```log
project statistics {
  commit_count: 167,
  storage_size: 459138266,
  repository_size: 289391998,
  wiki_size: 0,
  lfs_objects_size: 168066558,
  job_artifacts_size: 1679710,
  pipeline_artifacts_size: 0,
  packages_size: 0,
  snippets_size: 0,
  uploads_size: 21681491
}
```

**Actual behaviour**

> Property 'statistics' comes from an index signature, so it must be accessed with ['statistics'].

**Possible fixes**

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by NatoBoram** [comment]
Yep, exactly

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.23.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.23.0) :rocket:
